src/test/kotlin/blackjack/domain/participant/PlayerTest.kt

```diff
@@ -38,4 +41,35 @@ class PlayerTest {
         assertThat(result).isFalse()
     }
 
+    @Test
+    fun `플레이어가 블랙잭이 되면 카드를 더이상 받을 수 없습니다`() {
+        // given
+        val cards = mutableListOf(
+            Card.of(CardSuit.Heart, CardRank.Ten),
+            Card.of(CardSuit.Heart, CardRank.Nine),
+            Card.of(CardSuit.Heart, CardRank.Two),
+        )
+        val player = Dealer(cards = Cards(cards))
+
+        // when
+        val result = player.canReceiveCard()
+
+        // then
+        assertThat(result).isFalse()
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = [0, -1, -1000])
+    fun `플레이어의 배팅 금액이 0원 이하이면 에러를 던져야 한다`(bettingAmount: Int) {
+        // given
+
+        // when
+        val result = assertThrows(IllegalArgumentException::class.java) {
+            Player(name = "jay", bettingAmount = bettingAmount)
+        }
```

Comment:
AssertJ의 `assertThrows<...>`로 코틀린 인라인 함수를 활용하실 수 있습니다!
```suggestion
        val result = assertThrows<IllegalArgumentException> {
            Player(name = "jay", bettingAmount = bettingAmount)
        }
```

Reply:
[e6021a4](https://github.com/next-step/kotlin-blackjack/pull/795/commits/e6021a4c72575366bf1c22ab85f150fab48abdaf)